package.json

```diff
@@ -60,5 +44,14 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/diasdavid/js-peer-id.git"
+  },
+  "dignified": {
+    "webpack": {
+      "resolve": {
+        "alias": {
+          "node-forge": "../vendor/forge.bundle.js"
```

Comment:
This is currently only possible for `webpack`.  When the need arises we can add it for others as well